python/foglamp/common/plugin_discovery.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+"""Common Plugin Discovery Class"""
+
+import os
+from foglamp.common import logger
+
+__author__ = "Amarendra K Sinha"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+_logger = logger.setup(__name__)
+
+_FOGLAMP_DATA = os.getenv("FOGLAMP_DATA", default=None)
+_FOGLAMP_ROOT = os.getenv("FOGLAMP_ROOT", default='/usr/local/foglamp')
```

Comment:
can we find a common place for it? We also had discussed it when schedule module was done

Reply:
Please suggest.

Reply:
Can we have a common file  to import these 2? these are being used in server.py, scheduler.py and now here! and may be more...

Reply:
Let us do it via another ticket as it will require through testing of affected files.

Reply:
Pleae make a new common file, and import that here. we can fix others later.